code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -40,18 +40,9 @@
 /obj/item/weapon/legcuffs/bola/throw_impact(atom/hit_atom)
 	if(..() || !ishuman(hit_atom))//if it gets caught or the target can't be cuffed,
 		return//abort
-	var/mob/living/carbon/human/H = hit_atom
-	/*
-	var/num_legs
+	cacht(hit_atom)
 
-
-	for(var/limb_tag in H.species.bola_limbs)
-		var/obj/item/organ/external/E = H.has_organ_or_replacement(limb_tag)
-		if(E)
-			num_legs++
-
-	if(!H.legcuffed && num_legs >= 2)
-	*/
+/obj/item/weapon/legcuffs/bola/proc/cacht(var/mob/living/carbon/human/H)
```

Comment:
I dont see anything in here that determines escape time. A bola should be relatively easy to escape from, maybe 5 seconds of resisting.

Also does this work on necromorphs? It should work on any that have a bipedal form and both limbs, you may need to edit the hud data for necros to allow it, check the lurker for an example

Reply:
It works on necromorphs.

Reply:
![image](https://user-images.githubusercontent.com/49106866/111250537-cc95d380-860d-11eb-934c-19a48d778215.png)